src/model/model.ts

```diff
@@ -230,6 +248,24 @@ export class Model {
     return false;
   }
 
+  // removePolicies removes policy rules from the model.
+  public removePolicies(sec: string, ptype: string, rules: string[][]): boolean {
+    const ast = this.model.get(sec)?.get(ptype);
+    if (!ast) {
+      return true;
+    }
+
+    for (const rule of rules) {
+      if (this.hasPolicy(sec, ptype, rule)) {
+        ast.policy = _.filter(ast.policy, r => !util.arrayEquals(rule, r));
+      } else {
+        return false;
```

Comment:
ignore `return`,  same as `casbin-go`.

Reply:
Okay

Reply:
But, @nodece , when I remove ```return false;```, it will no longer remain a atomic function.

Reply:
You are right